include/SDL3_ttf/SDL_ttf.h

```diff
@@ -354,6 +354,9 @@ extern SDL_DECLSPEC bool SDLCALL TTF_GetFontDPI(TTF_Font *font, int *hdpi, int *
 /**
  * Font style flags
  */
+
+typedef Sint32 TTF_FontStyleFlags;
```

Comment:
If we're going to typedef this, it should be Uint32 and have more complete documentation, e.g.:
https://github.com/libsdl-org/SDL/blob/ec8a780ad47b397dc791bec2fa8bd307c9dbbc2c/include/SDL3/SDL_init.h#L64-L87